lighthouse-core/runner.js

```diff
@@ -87,6 +88,9 @@ class Runner {
         }
       }
 
+      // Make sure artifacts.Timing exists if we're running off of old artifacts
+      if (!artifacts.Timing) artifacts.Timing = [];
```

Comment:
do we need this? can just do `const timingEntries = (artifacts.Timing || []).concat(log.takeTimeEntries());` or whatever below.

We don't *need* to preserve the artifacts since we don't save them after this, but also no need to modify them (and slightly more straightforward to avoid it).

Reply:
sure, wfm